ui/tests/helpers/secret-engine/secret-engine-helpers.js

```diff
@@ -131,7 +165,12 @@ export const expectedValueOfConfigKeys = (type, string) => {
   }
 };
 
-export const fillInAwsConfig = async (withAccess = true, withAccessOptions = false, withLease = false) => {
+export const fillInAwsConfig = async (
+  withAccess = true,
+  withAccessOptions = false,
+  withLease = false,
+  withWif = false
```

Comment:
It seems like all these options are mutually exclusive, which tells me we should instead pass some sort of string to represent the state we want instead?

Reply:
would make it easier to read when the method is used! I'll amend.